hmda/src/main/scala/hmda/validation/rules/lar/quality/_2025/Q651_1.scala

```diff
@@ -0,0 +1,24 @@
+package hmda.validation.rules.lar.quality._2025
+
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.{ValidationFailure, ValidationResult}
+import hmda.validation.rules.EditCheck
+
+import scala.util.Try
+
+
+object Q651_1 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "Q651-1"
+
+  override def parent: String = "Q651"
+
+  override def apply(lar: LoanApplicationRegister): ValidationResult ={
+    when((lar.loan.combinedLoanToValueRatio is numeric) and
+      (lar.loan.combinedLoanToValueRatio.toFloat is lessThan(1.5) ) and
```

Comment:
you should be able to use `not numeric`.

Reply:
updated to reflect this suggestion